lib/cdo/rack/whitelist.rb

```diff
@@ -95,6 +95,7 @@ def call(env)
           request_cookies.keys.each do |key|
             response.add_header 'Vary', "X-COOKIE-#{key.tr('_', '-')}"
           end
+          response.add_header 'Vary', 'Cookie'
```

Comment:
I discovered this bug while running tests on local (development) server. The `Vary: Cookie` header is needed here in order for cached pages depending on the session cookie to not be cached by the browser when switching between signed-in sessions.